### DISCUSSION TITLE
hydration node mismatch with Vue

### DISCUSSION THREAD

**Discussion by royalswe** [vikejs/vike#859] [Discussion]
I'm trying to get the vue-router-v1 example working with fetched props from server to client but are stuck on hydration errors.
The error from console:
![image](https://user-images.githubusercontent.com/6486940/236234024-7d40369e-3559-4384-9803-04a4ba294373.png)

Here is a example repo which will give the error: https://github.com/royalswe/vite-plugin-ssr_vue-server-v1/tree/main/vue-router-v1/renderer

**Comment by brillout** [comment]
Hydration errors are usually user land issues. But let me know if you believe it's related to VPS.

In case you didn't see it already: https://vite-plugin-ssr.com/common-issues#hydration-mismatch.
